data/jotl/label/de.json

```diff
@@ -132,17 +132,147 @@
+    },
+    "298": {
+      ".": "Schlaflos",
+      "text": "Erleide in einer Runde, in der du eine lange Rase machst, Schaden durch einen Angriff."
+    },
+    "299": {
+      ".": "Masochistisch",
+      "text": "Beende das Szenario mit 3 oder weniger TP."
+    },
+    "300": {
+      ".": "Ausgenommen",
+      "text": "Töte ein Monster und erbeute in demselben Zug den fallengelassenen Münzmarker."
+    },
+    "301": {
+      ".": "Erstschlag",
+      "text": "Töte das erste Monster während des Szenarios."
+    },
+    "302": {
+      ".": "Pazifistisch",
+      "text": "Töte 3 oder weniger Monster."
+    },
+    "303": {
+      ".": "Profitorientiert",
+      "text": "Erbeute 1 oder mehr Münzmarker mir einer Beute-Aktion während du an 2 oder mehr Monster angrenzt."
+    },
+    "304": {
+      ".": "Nadelkissen",
+      "text": "Werde in einer Runde zum Angriffsziel von 3 oder mehr Monstern."
+    },
+    "305": {
+      ".": "Fußvolk",
+      "text": "Töte niemals ein Elite-Monster oder einen Boss."
+    },
+    "306": {
+      ".": "Trocken",
+      "text": "Benutze niemals Tränke."
+    },
+    "307": {
+      ".": "Verwüstung",
+      "text": "Führe 2 Aktionen mit Verlorenen-Symbol in demselben Zug aus."
+    },
+    "308": {
+      ".": "Unabhängig",
+      "text": "Beende deinen Zug niemals angrenzend an einen Verbündeten."
+    },
+    "309": {
+      ".": "Sadistisch",
+      "text": "Töte 5 oder mehr Monster."
+    },
+    "310": {
+      ".": "Ruhelos",
+      "text": "Mache keine lange Rast."
+    },
+    "311": {
+      ".": "Schatten",
+      "text": "Beende deinen Zug immer angrenzend an einen Verbündeten."
+    },
+    "312": {
+      ".": "Um die Ecke",
+      "text": "Töte, während du an ein Monster angrenzt, ein anderes Monster, das nicht an dich angrenzt."
+    },
+    "313": {
+      ".": "Spezialisiert",
+      "text": "Führe keine Standardaktion aus."
+    },
+    "314": {
+      ".": "Schlusslicht",
+      "text": "Mache keine kurze Rast."
+    },
+    "315": {
+      ".": "Hereingeplatzt",
+      "text": "Öffne eine Tür und beende denselben Zug in dem so aufgedeckten Raum."
+    },
+    "316": {
+      ".": "Ansteckend",
+      "text": "Wende einen negativen Zustand auf ein Monster an, während du unter einem negativen Zustand leidest."
+    },
+    "317": {
+      ".": "Ausgebrannt",
+      "text": "Werde als erster Charakter erschöpft."
+    }
+  },
   "custom": {
     "jotl": {
       "giant-viper": {
         "1": "Alle Angriffe an %data.monster.giant-viper% erhalten in dieser Runde Nachteil.",
         "2": "Füge +2 %game.action.attack% wenn das Ziel an einer %data.monster.giant-viper%'s Verbündeten angrenzt."
       },
       "black-sludge": {
-        "1": "Der nächste Gegner in Sichtline, ungachtet der Reichweite, erleidet %game.damage:[1+L/2{$math.ceil}]%."
+        "1": "Der nächste Gegner in Sichtlinie erleidet unabhängig der Reichweite %game.damage:[1+L/2{$math.ceil}]%."
```

Comment:
Is this the official wording? I want to keep original card content if possible. I am bit confused now, because we own German JOTL, but I can't remember if I looked up the original cards myself or not (and currently game not here), so just asking to be sure. Thanks!

Reply:
Yes, it's exactly the wording of the card. I'm owning the physical German version of JOTL from 2020 and as I know the 1st and only version.

Reply:
I have the same text in my german JOTL version like keinkurt.